package.json

```diff
@@ -2,21 +2,25 @@
   "name": "kitsu-wiki",
   "version": "0.1.0",
   "dependencies": {
-    "@apollo/client": "^3.2.7",
+    "@apollo/client": "^3.3.0",
     "@fortawesome/fontawesome-free": "^5.15.1",
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.14",
     "@types/node": "^14.14.20",
+    "@types/react-lazyload": "^3.0.0",
```

Comment:
please add to dev dependencies. Nothing typescript should be here.

Reply:
Ah, I'm not all that familiar with the difference between dev dependencies or not yet, will do